sources/web/datalab/polymer/modules/file-manager/file-manager.ts

```diff
@@ -130,6 +130,13 @@ abstract class DatalabFile {
   name: string;
   status?: DatalabFileStatus;
   type: DatalabFileType;
+
+  public getPreviewName(): string {
```

Comment:
Unless we're using this string for something else, can we make it `getPreviewElementName` and include the "-preview" part in it?